configure.ac

```diff
@@ -31,7 +31,7 @@ dnl your responsibility whether the result works or not.])])
 AC_COPYRIGHT([Copyright (c) 1996-2021, PostgreSQL Global Development Group])
 AC_CONFIG_SRCDIR([src/backend/access/common/heaptuple.c])
 AC_CONFIG_AUX_DIR(config)
-AC_PREFIX_DEFAULT(/usr/local/cbdb)
+AC_PREFIX_DEFAULT(/usr/local/cloudberry)
```

Comment:
This is a default path, not a word. Change it will make unnecessary troubles, please recover that.

Reply:
All right. has been updated in the latest commit https://github.com/apache/cloudberry/pull/929/commits/2e0efb7fce5f00ada10d0327c8ecd41e9462f3f7